app/code/Magento/Quote/Model/ResourceModel/Quote/QuoteIdMask.php

```diff
@@ -22,4 +22,27 @@ protected function _construct()
     {
         $this->_init('quote_id_mask', 'entity_id');
     }
+
+    /**
+     * Retrieves masked quote id
+     *
+     * Uses direct DB query due to performance reasons
+     *
+     * @param int $quoteId
+     * @return string|null
+     */
+    public function getMaskedQuoteId(int $quoteId): ?string
+    {
+        $connection = $this->getConnection();
+        $mainTable = $this->getMainTable();
+        $field = $connection->quoteIdentifier(sprintf('%s.%s', $mainTable, 'quote_id'));
+
+        $select = $connection->select()
+            ->from($mainTable, ['masked_id'])
+            ->where($field . '=?', $quoteId);
+
+        $result = $connection->fetchOne($select);
+
+        return $result ?: null;
```

Comment:
Same here, we can simplify it like the following
```suggestion
        return $connection->fetchOne($select) ?: null;
```

Reply:
Previous implementation looks more readable for me.
But it's my subjective opinion :)